parquet-hadoop/src/test/java/parquet/hadoop/TestInputFormat.java

```diff
@@ -105,17 +104,6 @@ public void testThrowExceptionWhenMaxSplitSizeIsSmallerThanMinSplitSize() throws
   }
 
   @Test
-  public void testThrowExceptionWhenMaxSplitSizeIsSmallerThanMinSplitSizeTaskSide() throws IOException {
-    try {
-      generateTSSplitByMinMaxSize(50, 49);
-      fail("should throw exception when max split size is smaller than the min split size");
-    } catch (ParquetDecodingException e) {
-      assertEquals("maxSplitSize and minSplitSize should be positive and max should be greater or equal to the minSplitSize: maxSplitSize = 49; minSplitSize is 50"
-              , e.getMessage());
-    }
-  }
```

Comment:
I've gone through and verified that split calculation is thoroughly tested. Because we no longer do the split calculations tested here, these tests are no longer useful. However, we do want to make sure that the relevant parts are tested:
* Mapping row groups to splits is very important and possibly impacted by changes to how splits are calculated. I've gone through and verified that the tests for this are thorough and still apply to using FileSplits. I'm confident that this doesn't need any more testing and is correct.
* Split calculation is now handled by FileInputFormat, so testing should be done upstream. We don't want to test this because it creates an unnecessary reliance on a library's behavior.